expat/coverage.sh

```diff
@@ -38,8 +38,11 @@ _get_coverage_dir() {
 _configure() {
     local configure_args=()
 
-    ${unicode_enabled} \
-            && configure_args+=( CPPFLAGS='-DXML_UNICODE -DXML_UNICODE_WCHAR_T' )
+    if [ "${unicode_enabled}" = true ]; then
+       configure_args+=( CPPFLAGS='-DXML_UNICODE -DXML_UNICODE_WCHAR_T -DXML_ATTR_INFO' )
+    else
+       configure_args+=( CPPFLAGS='-DXML_ATTR_INFO' )
+    fi
```

Comment:
We need yet another switch for the CI — e.g. `attr_info_enabled` — here, to cover both enabled and disabled.

Reply:
will change the passing xml-attr-info parameter to build & config so that the compilation take place for each case.